CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v10.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v10.0.0...HEAD)] - 2020-xx-xx
 
-- Write your awesome change here (GH-xxxx)
+- Fixed eks security group id outputs and additional security group id inputs (by @slimm609)
```

Comment:
This is a breaking change. You've changed the value of the `cluster_security_group_id` output.